openlibrary/templates/type/edition/view.html

```diff
@@ -168,7 +168,8 @@
 $if ebook_status == "read-online":
```

Comment:
Just a question -- does "read-online" ebook_status guarantee the page will have an ocaid (i.e. an archive.org identifier)?